layouts/shortcodes/deprecation/list.md

```diff
@@ -1,8 +1,18 @@
 {{ $deprecations := .Site.Data.deprecations }}
+{{ $majors := .Site.Data.majorReleases }}
 {{ $this := . }}
+{{ $eolDate := "FUBAR" }}
 {{ range $id, $dep := $deprecations }}
 ### [{{ $id }}] {{ $dep.name }} {#{{ $id }}}
-Beginning with the [{{ $dep.release }}](/docs/release_notes/{{ lower $dep.release }}) release, {{ $dep.summary }}
+<!-- this will access .Site.Data.majorReleases[ $dep.eol ] -->
+{{ $major := index $majors $dep.eol }}
+|  |  |
+| ---- | ---- |
+| **Introduced in release** | [{{ $dep.release }}](/docs/release_notes/{{ lower $dep.release }}) ({{ $dep.date }}) |
+| **Expected end of life** | [{{ if $dep.eol }}{{ $dep.eol }}{{ end }}](/docs/nodejs_support_policy) ({{ $major.date }}) |
+| **Status** | Active |
```

Comment:
*issue:* I'd imagine this would be set as part of the deprecation matadata itself. Currently it is hardcoded..

Reply:
Changing this to a "suggestion" instead. Spoke with Jamie and we are not going to have these change their status in the next year, so it is not a "blocker" per se. Will leave it up to him if he wants to do it now or not.

Reply:
leave as is